examples/website/point-cloud/app.tsx

```diff
@@ -31,15 +32,23 @@ const INITIAL_VIEW_STATE: OrbitViewState = {
 const transitionInterpolator = new LinearInterpolator(['rotationOrbit']);
 
 export default function App({
-  onLoad
+  onLoad,
+  device
 }: {
   onLoad?: (data: {count: number; progress: number}) => void;
+  device?: Device;
 }) {
   const [viewState, updateViewState] = useState<OrbitViewState>(INITIAL_VIEW_STATE);
   const [isLoaded, setIsLoaded] = useState<boolean>(false);
+  const [hasInteracted, setHasInteracted] = useState<boolean>(false);
 
   useEffect(() => {
-    if (!isLoaded) {
+    // we have to re-load the data on device change
```

Comment:
IIRC, I didn't have to make the demo apps aware of device switching in luma. The details are not fresh but I assume it is because the examples are recreated when device changes.

While we are making a few examples work this is OK, but once we start mass migration it is better to require less changes in the apps.